robotiq_force_torque_sensor/CMakeLists.txt

```diff
@@ -34,7 +41,10 @@ generate_messages(
 ## catkin specific configuration ##
 ###################################
 catkin_package(
-  CATKIN_DEPENDS message_runtime std_msgs
+  CATKIN_DEPENDS
+    message_runtime
+    std_msgs
+    launch_manager_interface
```

Comment:
```suggestion
```
I believe you should only require this if dependent projects need this package. In that case, those project should have an explicity dependency themselves